jupyter-extension/src/plugins/mount/customFileBrowser.ts

```diff
@@ -129,10 +130,86 @@ const createCustomFileBrowser = (
         },
       });
 
+      // We need to register this as an app command, but because this function is called multiple times we only want to register it once.
+      // This command must be registered as an app command to work with notification commandIds
+      if (!app.commands.hasCommand('open-pachyderm-sdk')) {
+        app.commands.addCommand('open-pachyderm-sdk', {
+          execute: () => {
+            window
+              ?.open('https://docs.pachyderm.com/latest/sdk/', '_blank')
+              ?.focus();
+          },
+        });
+      }
+
+      commands.addCommand('open-determined', {
```

Comment:
not sure the name here makes the most sense; this feature will have utility outside of determined too

Reply:
yeah good point, will switch it up!